relay-server/src/services/processor.rs

```diff
@@ -1300,12 +1317,6 @@ impl EnvelopeProcessorService {
             }
         };
 
-        if let Some(sampling_state) = state.sampling_project_state.clone() {
-            state
-                .envelope_mut()
-                .parametrize_dsc_transaction(&sampling_state.config.tx_name_rules);
-        }
```

Comment:
I moved this to `process_envelope`, because it should be done for any DSC, not just events.

Reply:
Please add such info as a code comment not just PR comments.

```
// Parameterize DSC for all enevelopes because ...
```